pkg/convert/tf.go

```diff
@@ -1107,6 +1107,18 @@ func convertObjectConsKeyExpr(state *convertState, inBlock bool,
 	scopes *scopes, fullyQualifiedPath string, expr *hclsyntax.ObjectConsKeyExpr,
 ) hclwrite.Tokens {
 	// Seems we can just ignore ForceNonLiteral here
+	switch exprWrapped := expr.Wrapped.(type) {
+	case *hclsyntax.TemplateExpr:
+		// For some reason these are template expressions, not just literals,
```

Comment:
> For some reason these are template expressions, not just literals,

I believe a template expression is a string that can have literal _parts_ (think interpolated strings) but if it is a single string then it is a template expression with a single part being the literal value